<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Module 4 Lab: Microsatellite Instability Visualization | Genomics for Precision Oncology 2025</title>
  <meta name="description" content="Workshop website for the 2025 Genomics for Precision Oncology Canadian Bioinformatics Workshop" />
  <meta name="generator" content="bookdown 0.44 and GitBook 2.6.7" />

  <meta property="og:title" content="Module 4 Lab: Microsatellite Instability Visualization | Genomics for Precision Oncology 2025" />
  <meta property="og:type" content="book" />
  <meta property="og:image" content="https://bioinformaticsdotca.github.io/GPO_Hal-2511/img/bioinformatics_logo.png" />
  <meta property="og:description" content="Workshop website for the 2025 Genomics for Precision Oncology Canadian Bioinformatics Workshop" />
  <meta name="github-repo" content="bioinformaticsdotca/GPO_Hal-2511" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Module 4 Lab: Microsatellite Instability Visualization | Genomics for Precision Oncology 2025" />
  
  <meta name="twitter:description" content="Workshop website for the 2025 Genomics for Precision Oncology Canadian Bioinformatics Workshop" />
  <meta name="twitter:image" content="https://bioinformaticsdotca.github.io/GPO_Hal-2511/img/bioinformatics_logo.png" />

<meta name="author" content="Faculty: Daniel Gaston, Victor Martinez" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon" />
<link rel="prev" href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "LearningResource",
  "name": "Genomics for Precision Oncology",
  "description": "The complete set of tutorials, slides, and practical exercises for the Genomics for Precision Oncology 2025 Canadian Bioinformatics Workshop, hosted on GitHub Pages.",
  "url": "https://bioinformaticsdotca.github.io/GPO_Hal-2511",
  "learningResourceType": ["tutorial", "slides", "exercise"],
  "inLanguage": "en",
  "license": "https://creativecommons.org/licenses/by/4.0/",
  "provider": {
    "@type": "Organization",
    "name": "Bioinformatics.ca",
    "url": "https://bioinformatics.ca"
  },
  "keywords": "",
  "isPartOf": {
    "@type": "Course",
    "@id": "https://bioinformatics.ca/workshops-all/2025-genomics-for-precision-oncology-halifax-ns/"
  }
}
</script>


<style type="text/css">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://bioinformaticsdotca.github.io/">
 <img src="img/bioinformatics_logo.png" width="90%" alt="bioinformatics.ca logo" style="display:block; margin-left:auto; margin-right:auto; margin-top:15px;">
 </a>
<li><center><strong><a href="./"><br>Genomics for Precision Oncology<br><br></a></strong></center></li>
<a href="./">
 <img src="img/CBW-GPO-cancer-logo-w-transparency-1024x1024.png" width="70%" alt="Genomics for Precision Oncology logo" style="display:block; margin-left:auto; margin-right:auto; ">
 </a>
<br>

<li class="divider"></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Workshop Info</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#pre-work"><i class="fa fa-check"></i>Pre-work</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#create-a-franklin-account"><i class="fa fa-check"></i>1. Create a Franklin Account</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#download-and-install-igv-integrative-genomics-viewer"><i class="fa fa-check"></i>2. Download and Install IGV (Integrative Genomics Viewer)</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#class-photo"><i class="fa fa-check"></i>Class Photo</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#schedule"><i class="fa fa-check"></i>Schedule</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="meet-your-faculty.html"><a href="meet-your-faculty.html"><i class="fa fa-check"></i>Meet Your Faculty</a>
<ul>
<li class="chapter" data-level="" data-path="meet-your-faculty.html"><a href="meet-your-faculty.html#instructors"><i class="fa fa-check"></i>Instructors</a></li>
<li class="chapter" data-level="" data-path="meet-your-faculty.html"><a href="meet-your-faculty.html#teaching-assistants"><i class="fa fa-check"></i>Teaching Assistants</a></li>
<li class="chapter" data-level="" data-path="meet-your-faculty.html"><a href="meet-your-faculty.html#facilitator"><i class="fa fa-check"></i>Facilitator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="data-and-compute-setup.html"><a href="data-and-compute-setup.html"><i class="fa fa-check"></i>Data and Compute Setup</a></li>
<li class="part"><span><b>II Modules</b></span></li>
<li class="chapter" data-level="" data-path="module-1.html"><a href="module-1.html"><i class="fa fa-check"></i>Module 1</a>
<ul>
<li class="chapter" data-level="" data-path="module-1.html"><a href="module-1.html#lecture"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="part"><span><b>III Modules</b></span></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html"><i class="fa fa-check"></i>Module 2</a>
<ul>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#lecture-1"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#lab-part-1"><i class="fa fa-check"></i>Lab (Part 1)</a>
<ul>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#aws-and-unix-basics"><i class="fa fa-check"></i>AWS and Unix Basics</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#docker"><i class="fa fa-check"></i>Docker</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#investigating-some-genomics-file-formats-and-metadata"><i class="fa fa-check"></i>Investigating Some Genomics File Formats and Metadata</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#lab-part-2---implementing-the-gatk-best-practices-workflow"><i class="fa fa-check"></i>Lab (Part 2) - Implementing the GATK Best Practices Workflow</a>
<ul>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#fastqc-quality-control"><i class="fa fa-check"></i>FastQC Quality Control</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#alignmentshort-read-mapping-with-bwa-mem"><i class="fa fa-check"></i>Alignment/Short-Read Mapping with BWA-MEM</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#gatk-pre-processing"><i class="fa fa-check"></i>GATK Pre-Processing</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#putting-everything-together"><i class="fa fa-check"></i>Putting Everything Together</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#viewing-bams-in-igv"><i class="fa fa-check"></i>Viewing BAMs in IGV</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#somatic-variant-calling-with-mutect2"><i class="fa fa-check"></i>Somatic Variant Calling with MuTect2</a></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#variant-annotation-with-vep"><i class="fa fa-check"></i>Variant Annotation with VEP</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-2.html"><a href="module-2.html#answers-to-guiding-questions"><i class="fa fa-check"></i>Answers to Guiding Questions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html"><i class="fa fa-check"></i>Module 3</a>
<ul>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#lecture-2"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#lab-clinical-variant-interpretation-with-franklin"><i class="fa fa-check"></i>Lab: Clinical Variant Interpretation with Franklin</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#learning-objectives"><i class="fa fa-check"></i>Learning Objectives</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#lab-overview"><i class="fa fa-check"></i>Lab Overview</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#part-1-account-creation-and-platform-navigation"><i class="fa fa-check"></i>Part 1: Account Creation and Platform Navigation</a>
<ul>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#creating-your-franklin-account"><i class="fa fa-check"></i>Creating Your Franklin Account</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#platform-navigation-understanding-the-main-interface"><i class="fa fa-check"></i>Platform Navigation: Understanding the Main Interface</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#creating-and-searching-for-variants"><i class="fa fa-check"></i>Creating and Searching for Variants</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#creating-new-cases"><i class="fa fa-check"></i>Creating New Cases</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#understanding-franklins-automated-classification-logic"><i class="fa fa-check"></i>Understanding Franklin’s Automated Classification Logic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#part-2-exploring-variant-classification"><i class="fa fa-check"></i>Part 2: Exploring Variant Classification</a>
<ul>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-1-searching-for-a-variant"><i class="fa fa-check"></i>Step 1: Searching for a Variant</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-2-understanding-the-classification-workspace"><i class="fa fa-check"></i>Step 2: Understanding the Classification Workspace</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-3-reviewing-automatically-assigned-acmg-codes"><i class="fa fa-check"></i>Step 3: Reviewing Automatically Assigned ACMG Codes</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-4-deep-dive-into-population-data-pm2-example"><i class="fa fa-check"></i>Step 4: Deep Dive into Population Data (PM2 Example)</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-5-editing-evidence-strength"><i class="fa fa-check"></i>Step 5: Editing Evidence Strength</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-6-exploring-additional-evidence-categories"><i class="fa fa-check"></i>Step 6: Exploring Additional Evidence Categories</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#part-3-uploading-and-classifying-your-case-vcf-45-minutes"><i class="fa fa-check"></i>Part 3: Uploading and Classifying Your Case VCF (45 minutes)</a>
<ul>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-1-download-your-case-data"><i class="fa fa-check"></i>Step 1: Download Your Case Data</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-2-create-a-new-case-in-franklin"><i class="fa fa-check"></i>Step 2: Create a New Case in Franklin</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#part-4-variant-interpretation"><i class="fa fa-check"></i>Part 4: Variant interpretation</a>
<ul>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-3-navigating-to-your-case"><i class="fa fa-check"></i>Step 3: Navigating to Your Case</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-4-reviewing-brca2-variants-in-your-case"><i class="fa fa-check"></i>Step 4: Reviewing BRCA2 Variants in Your Case</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-5-adding-patient-phenotype-information"><i class="fa fa-check"></i>Step 5: Adding Patient Phenotype Information</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-6-exploring-case-level-quality-control-and-variant-filtering"><i class="fa fa-check"></i>Step 6: Exploring Case-Level Quality Control and Variant Filtering</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-7-deep-dive-into-variant-classification-brca2c.1813dup"><i class="fa fa-check"></i>Step 7: Deep Dive into Variant Classification: BRCA2:c.1813dup</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#reviewing-the-franklin-acmg-classification"><i class="fa fa-check"></i>Reviewing the Franklin ACMG Classification</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#exploring-the-variant-assessment-tab"><i class="fa fa-check"></i>Exploring the Variant Assessment Tab</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#exploring-associated-conditions"><i class="fa fa-check"></i>Exploring Associated Conditions</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#using-the-publications-tab"><i class="fa fa-check"></i>Using the Publications Tab</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#gene-assessment-tab"><i class="fa fa-check"></i>Gene Assessment Tab</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#step-8-finalizing-your-classification-and-creating-a-clinical-report"><i class="fa fa-check"></i>Step 8: Finalizing Your Classification and Creating a Clinical Report</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#part-5-extension-exercise-optional"><i class="fa fa-check"></i>Part 5: Extension Exercise (Optional)</a></li>
<li class="chapter" data-level="" data-path="module-3.html"><a href="module-3.html#summary"><i class="fa fa-check"></i>Summary</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><i class="fa fa-check"></i>Module 4 Lab - Case 3: STK11 Deletion in Peutz-Jeghers Syndrome</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#learning-objectives-1"><i class="fa fa-check"></i>Learning Objectives</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#background"><i class="fa fa-check"></i>Background</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#peutz-jeghers-syndrome"><i class="fa fa-check"></i>Peutz-Jeghers Syndrome</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#the-stk11-gene-and-disease-mechanism"><i class="fa fa-check"></i>The STK11 Gene and Disease Mechanism</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#rationale-for-manual-review"><i class="fa fa-check"></i>Rationale for Manual Review</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-1-installing-the-integrative-genomics-viewer"><i class="fa fa-check"></i>Section 1: Installing the Integrative Genomics Viewer</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#overview-of-igv"><i class="fa fa-check"></i>Overview of IGV</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-1-download-igv"><i class="fa fa-check"></i>Step 1: Download IGV</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#windows-systems"><i class="fa fa-check"></i>Windows Systems</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#macos-systems"><i class="fa fa-check"></i>macOS Systems</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-2-installation-procedures"><i class="fa fa-check"></i>Step 2: Installation Procedures</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#windows-installation"><i class="fa fa-check"></i>Windows Installation</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#macos-installation"><i class="fa fa-check"></i>macOS Installation</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-3-installation-verification"><i class="fa fa-check"></i>Step 3: Installation Verification</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#troubleshooting-common-issues"><i class="fa fa-check"></i>Troubleshooting Common Issues</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#java-runtime-not-found"><i class="fa fa-check"></i>Java Runtime Not Found</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#macos-gatekeeper-restrictions"><i class="fa fa-check"></i>macOS Gatekeeper Restrictions</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#application-launch-failure"><i class="fa fa-check"></i>Application Launch Failure</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-2-acquiring-case-study-data-files"><i class="fa fa-check"></i>Section 2: Acquiring Case Study Data Files</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#binary-alignment-map-bam-files-and-indices"><i class="fa fa-check"></i><strong>Binary Alignment Map (BAM) Files and Indices</strong></a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#browser-extensible-data-bed-annotation-tracks"><i class="fa fa-check"></i><strong>Browser Extensible Data (BED) Annotation Tracks</strong></a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-1-accessing-data-files-via-jupyter-notebook"><i class="fa fa-check"></i>Step 1: Accessing Data Files via Jupyter Notebook</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-2-local-directory-organization"><i class="fa fa-check"></i>Step 2: Local Directory Organization</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-3-downloading-files-from-jupyter-to-your-local-machine"><i class="fa fa-check"></i>Step 3: Downloading Files from Jupyter to Your Local Machine</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-3-loading-data-into-igv-and-navigating-to-stk11"><i class="fa fa-check"></i>Section 3: Loading Data into IGV and Navigating to STK11</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-1-verifying-genome-build-selection"><i class="fa fa-check"></i>Step 1: Verifying Genome Build Selection</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-2-loading-bam-alignment-files"><i class="fa fa-check"></i>Step 2: Loading BAM Alignment Files</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-3-loading-bed-annotation-files"><i class="fa fa-check"></i>Step 3: Loading BED Annotation Files</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-4-navigating-to-the-stk11-gene-region"><i class="fa fa-check"></i>Step 4: Navigating to the STK11 Gene Region</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-5-normalizing-coverage-scale-across-samples"><i class="fa fa-check"></i>Step 5: Normalizing Coverage Scale Across Samples</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-6-focused-navigation-to-the-deletion-region"><i class="fa fa-check"></i>Step 6: Focused Navigation to the Deletion Region</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-7-verification"><i class="fa fa-check"></i>Step 7: Verification</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#common-issues-and-troubleshooting"><i class="fa fa-check"></i>Common Issues and Troubleshooting</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-4-quality-assessment-of-the-suspected-deletion"><i class="fa fa-check"></i>Section 4: Quality Assessment of the Suspected Deletion</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-1-configuring-igv-display-settings-for-coverage-analysis"><i class="fa fa-check"></i>Step 1: Configuring IGV Display Settings for Coverage Analysis</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-2-quantitative-coverage-analysis"><i class="fa fa-check"></i>Step 2: Quantitative Coverage Analysis</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-3-mapping-quality-assessment"><i class="fa fa-check"></i>Step 3: Mapping Quality Assessment</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-4-deletion-breakpoint-definition"><i class="fa fa-check"></i>Step 4: Deletion Breakpoint Definition</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-5-assessment-of-problematic-genomic-regions"><i class="fa fa-check"></i>Step 5: Assessment of Problematic Genomic Regions</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-6-evaluation-of-supporting-genetic-evidence"><i class="fa fa-check"></i>Section 6: Evaluation of Supporting Genetic Evidence</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-7-integrating-evidence-and-reaching-a-quality-assessment-conclusion"><i class="fa fa-check"></i>Section 7: Integrating Evidence and Reaching a Quality Assessment Conclusion</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#proceeding-to-clinical-variant-interpretation"><i class="fa fa-check"></i>Proceeding to Clinical Variant Interpretation</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-5-clinical-interpretation-of-copy-number-loss-using-clingen-guidelines"><i class="fa fa-check"></i>Section 5: Clinical Interpretation of Copy Number Loss Using ClinGen Guidelines</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#overview-of-clingen-cnv-interpretation-framework"><i class="fa fa-check"></i>Overview of ClinGen CNV Interpretation Framework</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#accessing-the-clingen-cnv-loss-calculator"><i class="fa fa-check"></i>Accessing the ClinGen CNV Loss Calculator</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-1---initial-assessment-of-genomic-content"><i class="fa fa-check"></i>Section 1 - Initial Assessment of Genomic Content</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#criterion-1a-does-the-cnv-include-any-protein-coding-or-other-critical-genomic-elements"><i class="fa fa-check"></i>Criterion 1A: Does the CNV include any protein-coding or other critical genomic elements?</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-2---overlap-with-establishedpredicted-hi-or-established-benign-genesgenomic-regions"><i class="fa fa-check"></i><strong>Section 2 - Overlap with Established/Predicted HI or Established Benign Genes/Genomic Regions</strong></a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#criterion-2a-does-the-cnv-completely-overlap-an-established-hi-gene-or-genomic-region"><i class="fa fa-check"></i><strong>Criterion 2A: Does the CNV completely overlap an established HI gene or genomic region?</strong></a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#criterion-2b-2e-overlap-with-establishedpredicted-hi-or-established-benign-genesgenomic-regions"><i class="fa fa-check"></i><strong>Criterion 2B-2E: Overlap with Established/Predicted HI or Established Benign Genes/Genomic Regions</strong></a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#criterion-2f-g-overlap-with-establishedpredicted-hi-or-established-benign-genesgenomic-regions"><i class="fa fa-check"></i><strong>Criterion 2F-G: Overlap with Established/Predicted HI or Established Benign Genes/Genomic Regions</strong></a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-3---evaluation-of-gene-number"><i class="fa fa-check"></i><strong>Section 3 - Evaluation of Gene Number</strong></a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-4---detailed-evaluation-of-genomic-content-using-published-literature-public-databases-andor-internal-lab-data"><i class="fa fa-check"></i><strong>Section 4 - Detailed Evaluation of Genomic Content Using Published Literature, Public Databases, and/or Internal Lab Data</strong></a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#section-5---evaluation-of-inheritance-patternfamily-history-for-patient-being-studied"><i class="fa fa-check"></i>Section 5 - Evaluation of Inheritance Pattern/Family History for Patient Being Studied</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#criterion-5h-inheritance-information-unavailable-or-uninformative-with-highly-specific-consistent-phenotype"><i class="fa fa-check"></i>Criterion 5H: Inheritance information unavailable or uninformative, with highly specific consistent phenotype</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-6-calculating-the-final-classification"><i class="fa fa-check"></i>Step 6: Calculating the Final Classification</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#step-7-clinical-recommendations-and-validation"><i class="fa fa-check"></i>Step 7: Clinical Recommendations and Validation</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#references"><i class="fa fa-check"></i>References</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#primary-literature"><i class="fa fa-check"></i>Primary Literature</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#software-and-databases"><i class="fa fa-check"></i>Software and Databases</a></li>
<li class="chapter" data-level="" data-path="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html"><a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html#guidelines-and-standards"><i class="fa fa-check"></i>Guidelines and Standards</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html"><i class="fa fa-check"></i>Module 4 Lab: Microsatellite Instability Visualization</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#learning-objectives-2"><i class="fa fa-check"></i>Learning Objectives</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#lab-overview-1"><i class="fa fa-check"></i>Lab Overview</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#background-understanding-microsatellite-instability"><i class="fa fa-check"></i>Background: Understanding Microsatellite Instability</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#part-1-data-download-from-jupyter-notebook"><i class="fa fa-check"></i>Part 1: Data Download from Jupyter Notebook</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#accessing-the-data-directory"><i class="fa fa-check"></i>Accessing the Data Directory</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#downloading-the-bam-files"><i class="fa fa-check"></i>Downloading the BAM Files</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#part-2-loading-data-into-igv"><i class="fa fa-check"></i>Part 2: Loading Data into IGV</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#installing-and-launching-igv"><i class="fa fa-check"></i>Installing and Launching IGV</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#loading-the-bam-files"><i class="fa fa-check"></i>Loading the BAM Files</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#part-3-visualizing-microsatellite-instability"><i class="fa fa-check"></i>Part 3: Visualizing Microsatellite Instability</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#step-1-navigate-to-the-microsatellite-locus"><i class="fa fa-check"></i>Step 1: Navigate to the Microsatellite Locus</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#step-2-configure-igv-track-display-settings"><i class="fa fa-check"></i>Step 2: Configure IGV Track Display Settings</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#step-3-observe-and-compare-msi-patterns"><i class="fa fa-check"></i>Step 3: Observe and Compare MSI Patterns</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#step-4-understanding-what-youre-seeing"><i class="fa fa-check"></i>Step 4: Understanding What You’re Seeing</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#step-5-lab-deliverable"><i class="fa fa-check"></i>Step 5: Lab Deliverable</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#summary-and-clinical-significance"><i class="fa fa-check"></i>Summary and Clinical Significance</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#clinical-applications-of-msi-testing"><i class="fa fa-check"></i>Clinical Applications of MSI Testing</a></li>
<li class="chapter" data-level="" data-path="module-4-lab-microsatellite-instability-visualization.html"><a href="module-4-lab-microsatellite-instability-visualization.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<h1 id="sponsors">Sponsors</h1>
<center>
<a href="https://www.cihr-irsc.gc.ca"><img src="img/sponsors/cihr.png" width=90% style="padding-bottom: 20px" alt="Canadian Institutes of Health Research logo."></a>

<a href="https://genomecanada.ca/"><img src="img/sponsors/genomecanada.png" height="55" style="display:inline-block;"  style="padding-bottom: 20px" alt="Genome Canada logo.">
<a href="https://www.ontariogenomics.ca/"><img src="img/sponsors/ontariogenomics.png" height="55" style="display:inline-block;" style="padding-bottom: 20px;" alt="Ontario Genomics logo.">
<br><br>
<a href="https://oicr.on.ca/"><img src="img/sponsors/oicr.png" width=35% style="display:inline-block;" style="padding-bottom: 20px" alt="Ontario Institute for Cancer Research logo.">
<a href="https://oicr.on.ca/"><img src="img/sponsors/hpc4health.png" width=55% style="display:inline-block;" style="padding-bottom: 20px" alt="HPC4Health logo.">
<a href="https://oicr.on.ca/"><img src="img/sponsors/BHCRI_logo-transparent.png" width=90% style="display:inline-block;" style="padding-bottom: 20px" alt="Beatrice Hunter Cancer Research Institute logo.">
<a href="https://oicr.on.ca/"><img src="img/sponsors/dal-logo-horizontal-colour.png" width=70% style="display:inline-block;" style="padding-bottom: 20px" alt="Dalhousie University logo.">

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genomics for Precision Oncology 2025</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="module-4-lab-microsatellite-instability-visualization" class="section level1 hasAnchor">
<h1>Module 4 Lab: Microsatellite Instability Visualization<a href="module-4-lab-microsatellite-instability-visualization.html#module-4-lab-microsatellite-instability-visualization" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Date: November 5, 2025
Summary: The lab focuses on visualizing microsatellite instability (MSI) patterns in tumor versus normal tissue using IGV, emphasizing the significance of allelic heterogeneity and its clinical applications, including Lynch syndrome screening and immunotherapy selection. Participants will download and analyze BAM files, observe instability at specific microsatellite loci, and understand the implications of MSI in cancer diagnostics and treatment.
Text: Microsatellite Instability Visualization Lab Overview</p>
<div id="learning-objectives-2" class="section level2 hasAnchor">
<h2>Learning Objectives<a href="module-4-lab-microsatellite-instability-visualization.html#learning-objectives-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>By the end of this lab, you will:</p>
<ol style="list-style-type: decimal">
<li>Visualize and interpret microsatellite instability patterns in tumor versus normal tissue using IGV</li>
<li>Recognize allelic heterogeneity as the molecular signature of mismatch repair deficiency</li>
<li>Connect MSI testing to clinical applications including Lynch syndrome screening and immunotherapy selection</li>
</ol>
</div>
<div id="lab-overview-1" class="section level2 hasAnchor">
<h2>Lab Overview<a href="module-4-lab-microsatellite-instability-visualization.html#lab-overview-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong>Duration:</strong> 20 minutes</p>
<p>Microsatellite instability (MSI) is a hypermutator phenotype caused by defective DNA mismatch repair. MSI-high tumors accumulate insertion and deletion mutations at repetitive DNA sequences (microsatellites), creating a distinctive molecular signature visible in sequencing data.</p>
<p>This lab uses data from a colorectal cancer case with confirmed MSI-high status (See reference at the end). You’ll visualize specific microsatellite loci where the tumor shows allelic heterogeneity compared to the patient’s normal tissue.</p>
</div>
<div id="background-understanding-microsatellite-instability" class="section level2 hasAnchor">
<h2>Background: Understanding Microsatellite Instability<a href="module-4-lab-microsatellite-instability-visualization.html#background-understanding-microsatellite-instability" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Microsatellites are repetitive DNA sequences (1-6 base pair motifs) that constitute approximately 3% of the human genome. Common types include mononucleotide repeats like (A)n, dinucleotide repeats like (CA)n, and higher-order repeats.</p>
<p>During DNA replication, polymerase can slip when copying repetitive sequences, creating insertion or deletion errors. In normal cells, the mismatch repair (MMR) system (comprising MLH1, MSH2, MSH6, and PMS2 proteins) recognizes and corrects these errors, reducing microsatellite mutation rates by 100-1000 fold.</p>
<p>When MMR is deficient (via germline pathogenic variants in Lynch syndrome or somatic MLH1 hypermethylation in sporadic tumors), polymerase slippage errors accumulate uncorrected across thousands of microsatellites genome-wide. This creates a hypermutator phenotype with mutation rates 100-1000 times higher than microsatellite-stable tumors.</p>
<p>Unlike typical clonal driver mutations, MSI manifests as multiple different indel alleles at each microsatellite locus. Each tumor subclone independently acquires random polymerase slippage errors, creating a characteristic “pile-up” appearance in sequencing data with insertions, deletions, and varying allele fractions. Normal tissue shows uniform read alignment because MMR efficiently corrects rare errors.</p>
</div>
<div id="part-1-data-download-from-jupyter-notebook" class="section level2 hasAnchor">
<h2>Part 1: Data Download from Jupyter Notebook<a href="module-4-lab-microsatellite-instability-visualization.html#part-1-data-download-from-jupyter-notebook" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="accessing-the-data-directory" class="section level3 hasAnchor">
<h3>Accessing the Data Directory<a href="module-4-lab-microsatellite-instability-visualization.html#accessing-the-data-directory" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li><p>Open your JupyterHub session in a web browser</p></li>
<li><p>Navigate to the course data directory:</p>
<pre><code>Module4/Microsatellite/</code></pre></li>
<li><p>Verify you can see the following files:</p>
<ul>
<li><code>test.normal.bam</code></li>
<li><code>test.normal.bam.bai</code></li>
<li><code>test.tumor.bam</code></li>
<li><code>test.tumor.bam.bai</code></li>
</ul></li>
</ol>
</div>
<div id="downloading-the-bam-files" class="section level3 hasAnchor">
<h3>Downloading the BAM Files<a href="module-4-lab-microsatellite-instability-visualization.html#downloading-the-bam-files" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Download the tumor sample:</strong></p>
<ol style="list-style-type: decimal">
<li>Right-click on <code>test.tumor.bam</code></li>
<li>Select <strong>“Download”</strong></li>
<li>Save to a location you can easily access (e.g., Desktop or Downloads folder)</li>
</ol>
<p><strong>Download the tumor index:</strong></p>
<ol style="list-style-type: decimal">
<li>Right-click on <code>test.tumor.bam.bai</code></li>
<li>Select <strong>“Download”</strong></li>
<li>Save to the <strong>same folder</strong> as the tumor BAM file</li>
</ol>
<p><strong>Download the normal sample:</strong></p>
<ol style="list-style-type: decimal">
<li>Right-click on <code>test.normal.bam</code></li>
<li>Select <strong>“Download”</strong></li>
<li>Save to the <strong>same folder</strong> as the tumor files</li>
</ol>
<p><strong>Download the normal index:</strong></p>
<ol style="list-style-type: decimal">
<li>Right-click on <code>test.normal.bam.bai</code></li>
<li>Select <strong>“Download”</strong></li>
<li>Save to the <strong>same folder</strong> as the other files</li>
</ol>
<aside>
<p>⚠️<strong>Requirement:</strong> The BAI index files must be in the same directory as their corresponding BAM files. IGV will not load alignments without the index files present.</p>
</aside>
<p><strong>File organization check:</strong></p>
<p>Your download folder should now contain exactly four files:</p>
<pre><code>test.tumor.bam
test.tumor.bam.bai
test.normal.bam
test.normal.bam.bai</code></pre>
</div>
</div>
<div id="part-2-loading-data-into-igv" class="section level2 hasAnchor">
<h2>Part 2: Loading Data into IGV<a href="module-4-lab-microsatellite-instability-visualization.html#part-2-loading-data-into-igv" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="installing-and-launching-igv" class="section level3 hasAnchor">
<h3>Installing and Launching IGV<a href="module-4-lab-microsatellite-instability-visualization.html#installing-and-launching-igv" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>If you haven’t already installed IGV from a previous lab session, refer to the previous case for installation instructions.</p>
<p><strong>Quick launch:</strong></p>
<ol style="list-style-type: decimal">
<li>Open IGV on your computer</li>
<li>Wait for the application to fully load (you should see the reference genome selector in the top left)</li>
</ol>
<p>Ensure you’re using the hg19 reference genome to match the alignment.</p>
<ol style="list-style-type: decimal">
<li>In the top left corner, locate the genome dropdown menu</li>
<li>If it doesn’t already show <strong>“Human hg19”</strong>, click the dropdown</li>
<li>Select <strong>“Human (hg19)”</strong> from the list</li>
<li>Wait for IGV to load the reference genome (this may take 10-15 seconds)</li>
</ol>
</div>
<div id="loading-the-bam-files" class="section level3 hasAnchor">
<h3>Loading the BAM Files<a href="module-4-lab-microsatellite-instability-visualization.html#loading-the-bam-files" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Load the normal sample:</strong></p>
<ol style="list-style-type: decimal">
<li>Click <strong>“File”</strong> → <strong>“Load from File…”</strong></li>
<li>Navigate to your download folder</li>
<li>Select <code>test.normal.bam</code></li>
<li>Click <strong>“Open”</strong></li>
<li>IGV will automatically detect and use the corresponding <code>.bai</code> index file</li>
</ol>
<p><strong>Load the tumor sample:</strong></p>
<ol style="list-style-type: decimal">
<li>Click <strong>“File”</strong> → <strong>“Load from File…”</strong> again</li>
<li>Select <code>test.tumor.bam</code></li>
<li>Click <strong>“Open”</strong></li>
</ol>
<p>You should now see two tracks in the IGV viewer:</p>
<ul>
<li><code>test.normal.bam</code> (top track)</li>
<li><code>test.tumor.bam</code> (bottom track)</li>
</ul>
<aside>
<p>💡 <strong>Tip:</strong> You can reorder tracks by clicking and dragging the track name labels on the left side. Keep normal on top and tumor on bottom for easier comparison.</p>
</aside>
</div>
</div>
<div id="part-3-visualizing-microsatellite-instability" class="section level2 hasAnchor">
<h2>Part 3: Visualizing Microsatellite Instability<a href="module-4-lab-microsatellite-instability-visualization.html#part-3-visualizing-microsatellite-instability" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>You’ll now navigate to a highly MSI-sensitive locus on chromosome 1 where the tumor shows instability compared to normal tissue.</p>
<div id="step-1-navigate-to-the-microsatellite-locus" class="section level3 hasAnchor">
<h3>Step 1: Navigate to the Microsatellite Locus<a href="module-4-lab-microsatellite-instability-visualization.html#step-1-navigate-to-the-microsatellite-locus" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>In IGV’s search box at the top of the screen, paste the coordinates below:</p>
<ul>
<li><code>chr1:16265055-16265085</code></li>
</ul>
<p>Press <strong>Enter</strong> to jump to the locus. You should see the reference sequence displaying:</p>
<ul>
<li><strong>Left flank:</strong> AAAGC</li>
<li><strong>Microsatellite:</strong> T repeated 19 times (T×19)</li>
<li><strong>Right flank:</strong> CATTC</li>
</ul>
<aside>
<p>💡 Confirming the flanking sequences (AAAGC on the left, CATTC on the right) ensures you’re viewing the correct microsatellite locus. This T×19 mononucleotide tract is one of 19 consecutive thymine bases in the reference genome.</p>
</aside>
</div>
<div id="step-2-configure-igv-track-display-settings" class="section level3 hasAnchor">
<h3>Step 2: Configure IGV Track Display Settings<a href="module-4-lab-microsatellite-instability-visualization.html#step-2-configure-igv-track-display-settings" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To optimize visualization of microsatellite instability patterns, adjust the alignment track settings:</p>
<p><strong>For both the tumour and normal BAM tracks:</strong></p>
<ol style="list-style-type: decimal">
<li><strong>Right-click</strong> on the alignment track name (either <code>test.tumor.bam</code> or <code>test.normal.bam</code>) and select <strong>“Expanded”</strong> view mode (if not already selected)</li>
<li><strong>Turn on “Show center line”</strong>
<ul>
<li>This helps visualize read continuity and gaps</li>
</ul></li>
</ol>
<aside>
<p>⚠️<strong>IGV performance note:</strong> If IGV becomes slow with downsampling disabled, you can re-enable it at a higher threshold (e.g., 1000 reads instead of the default 50). The key is to see enough reads to observe the heterogeneous indel pattern in the tumor.</p>
</aside>
</div>
<div id="step-3-observe-and-compare-msi-patterns" class="section level3 hasAnchor">
<h3>Step 3: Observe and Compare MSI Patterns<a href="module-4-lab-microsatellite-instability-visualization.html#step-3-observe-and-compare-msi-patterns" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Now examine the alignment patterns in the T×19 microsatellite region. Focus on the differences between tumor and normal tissue.</p>
<p><strong>Normal tissue track (test.normal.bam):</strong></p>
<ul>
<li>Reads align smoothly through the poly-T tract</li>
<li><strong>Little to no indel pile-up:</strong> You may see 1-2 isolated indel marks (sequencing errors), but no clustering</li>
<li>All reads show the same T×19 structure</li>
<li>Grey bars indicate reads matching the reference genome</li>
</ul>
<p><strong>Tumor tissue track (test.tumor.bam):</strong></p>
<ul>
<li><strong>Pile-up of deletions:</strong> Look for clustered <strong>colored tick marks</strong> (typically black or dark) within the T-run
<ul>
<li>These indicate multiple reads with deletions relative to the reference</li>
<li>The ticks appear as short vertical lines interrupting the grey read bars</li>
</ul></li>
<li><strong>Variable indel sizes:</strong> Different reads show different deletion lengths
<ul>
<li>Some reads may have -1 T deletion (18 Ts instead of 19)</li>
<li>Others may have -2, -3, or larger deletions</li>
<li>You may also see occasional insertions (less common than deletions)</li>
</ul></li>
<li><strong>Heterogeneous appearance:</strong> The “noisy” pattern reflects multiple tumor subclones with independent slippage errors</li>
<li><strong>White gaps in reads:</strong> Deletions appear as empty spaces where bases are missing (connected by a black line, and the number of nucleotides involved.</li>
</ul>
</div>
<div id="step-4-understanding-what-youre-seeing" class="section level3 hasAnchor">
<h3>Step 4: Understanding What You’re Seeing<a href="module-4-lab-microsatellite-instability-visualization.html#step-4-understanding-what-youre-seeing" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Why mononucleotide tracts are MSI-sensitive:</strong></p>
<p>Poly-T tracts like this T×19 microsatellite are the <strong>most unstable</strong> repeat type in MMR-deficient tumors:</p>
<ul>
<li>DNA polymerase frequently slips during replication of long homopolymer runs</li>
<li>Each slippage event creates a small insertion or deletion loop</li>
<li>Without functional MMR, these errors accumulate uncorrected</li>
<li>Each tumor cell lineage acquires different random slippage errors → allelic heterogeneity</li>
</ul>
<p><strong>Why you see deletion bias:</strong></p>
<p>At most mononucleotide repeats, deletions outnumber insertions due to the mechanics of polymerase slippage:</p>
<ul>
<li><strong>Template strand looping</strong> (forward slippage) creates deletions more frequently</li>
<li><strong>Nascent strand looping</strong> (backward slippage) creates insertions less frequently</li>
</ul>
<p><strong>Clinical significance of this single locus:</strong></p>
<p>Even observing instability at this <strong>one</strong> microsatellite strongly suggests genome-wide MMR deficiency. In MSI-high tumors, thousands of similar loci show comparable patterns across the genome.</p>
</div>
<div id="step-5-lab-deliverable" class="section level3 hasAnchor">
<h3>Step 5: Lab Deliverable<a href="module-4-lab-microsatellite-instability-visualization.html#step-5-lab-deliverable" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Take a screenshot of your IGV view showing both tracks</strong> (tumor and normal) with the microsatellite region visible. Your screenshot should clearly show:</p>
<ul>
<li>The chromosome 1 coordinates in the search box</li>
<li>Both <code>test.normal.bam</code> and <code>test.tumor.bam</code> tracks</li>
<li>The contrasting patterns: uniform alignment in normal vs. indel pile-up in tumor</li>
<li>The reference sequence track (if visible) showing the T×19 repeat</li>
</ul>
<p><strong>To capture the screenshot:</strong></p>
<ul>
<li><strong>Windows:</strong> Use the Snipping Tool or press Windows+Shift+S</li>
<li><strong>Mac:</strong> Press Command+Shift+4 and drag to select the IGV window</li>
<li><strong>Linux:</strong> Use Screenshot utility or press PrtScn</li>
</ul>
<p>Save the screenshot with a descriptive filename (e.g., <code>MSI_chr1_T19_comparison.png</code>)</p>
</div>
</div>
<div id="summary-and-clinical-significance" class="section level2 hasAnchor">
<h2>Summary and Clinical Significance<a href="module-4-lab-microsatellite-instability-visualization.html#summary-and-clinical-significance" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>By comparing tumor and normal alignments at microsatellite loci, you’ve visualized:</p>
<ol style="list-style-type: decimal">
<li><strong>Allelic heterogeneity:</strong> Multiple different indel patterns at the same genomic position</li>
<li><strong>Tumor-specific instability:</strong> Normal tissue maintains stable repeat lengths, while tumor tissue shows widespread variation</li>
<li><strong>Frameshift accumulation:</strong> Insertions and deletions create reading frame disruptions in coding sequences</li>
</ol>
<div id="clinical-applications-of-msi-testing" class="section level3 hasAnchor">
<h3>Clinical Applications of MSI Testing<a href="module-4-lab-microsatellite-instability-visualization.html#clinical-applications-of-msi-testing" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>1. Diagnostic classification:</strong></p>
<ul>
<li>MSI-high (MSI-H): Instability at ≥30% of tested loci</li>
<li>MSI-low (MSI-L): Instability at &lt;30% of loci</li>
<li>Microsatellite stable (MSS): No instability detected</li>
</ul>
<p><strong>2. Lynch syndrome screening:</strong></p>
<ul>
<li>MSI-high tumors in young patients suggest germline mismatch repair defects</li>
<li>Triggers reflex testing for MLH1, MSH2, MSH6, PMS2 mutations</li>
</ul>
<p><strong>3. Immunotherapy selection:</strong></p>
<ul>
<li>MSI-high tumors are highly responsive to immune checkpoint inhibitors (anti-PD-1/PD-L1)</li>
<li>FDA-approved indication: pembrolizumab for MSI-H/dMMR solid tumors</li>
<li>Response rates: 40-60% in MSI-H colorectal cancer vs. &lt;5% in MSS tumors</li>
</ul>
<p><strong>4. Prognostic stratification:</strong></p>
<ul>
<li>MSI-high colorectal cancers have better stage-adjusted prognosis</li>
<li>May not benefit from 5-fluorouracil chemotherapy (standard for MSS tumors)</li>
</ul>
</div>
<div id="references-1" class="section level3 hasAnchor">
<h3>References<a href="module-4-lab-microsatellite-instability-visualization.html#references-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Ziegler, J., Hechtman, J.F., Rana, S. et al. A deep multiple instance learning framework improves microsatellite instability detection from tumor next generation sequencing. <em>Nat Commun</em> 16, 136 (2025).</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="module-4-lab---case-3-stk11-deletion-in-peutz-jeghers-syndrome.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": false,
    "twitter": false,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/bioinformaticsdotca/GPO_Hal-2511/blob/main/040-module-4.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["_main.pdf", "_main.epub"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
